tests/auto_shift/test_auto_shift.cpp

```diff
@@ -68,3 +68,30 @@ TEST_F(AutoShift, key_release_after_timeout) {
     run_one_scan_loop();
     VERIFY_AND_CLEAR(driver);
 }
+
+// test auto shift and space cadet interaction
+// press shift, press key, release shift, release key
+// the right interaction is we only get the shifted key
+// the wrong interaction is we get a bracket and a shifted key
+TEST_F(AutoShift, auto_shift_with_space_cadet) {
+    TestDriver driver;
+    InSequence s;
+    auto       left_shift = KeymapKey(0, 0, 0, SC_LSPO);
+    auto       key_a      = KeymapKey(0, 1, 0, KC_A);
+
+    set_keymap({regular_key});
```

Comment:
`regular_key` doesn't exist here. This should be: 
```suggestion
    set_keymap({left_shift, key_a});
```